src/components/cardGrid/cardGrid.component.jsx

```diff
@@ -0,0 +1,38 @@
+import Card from "../card/card.component";
+import "./cardGrid.style.css";
+/**
+ * Renders a grid of cards based on the provided data.
+ *
+ * @component
+ * @param {Object} props - The component properties.
+ * @param {Array} props.cards - An array of card objects to be rendered in the grid
+ * @param {string} [props.fallbackText="No Data Available Yet!"] - The text to display when no cards are available.
+ *  @param {"small" | "big"} props.gridSize -The size of gird
+ *  @param {"small" | "big"} props.cardSize -The size of cards
+ * @returns {JSX.Element} - The CardGrid component.
+ *
+ * @typedef {Object} Card
+ * @property {string} id - The unique identifier for the card.
+ * @property {string} title - The title of the card.
+ * @property {Object} data - The description data of card.
```

Comment:
![image](https://github.com/ASU-Semicolon/The-New-Dashboard/assets/80486237/a31d37ec-d128-4c1d-9bcd-9b80907bff14)
Vscode gives the type of `cards` as `any[]`
we need to let others know the structure of this type